blob/blob.go

```diff
@@ -41,6 +41,17 @@
 // Alternatively, you can construct a *Bucket using blob.Open by providing
 // a URL that's supported by a blob subpackage that you have linked
 // in to your application.
+//
+//
+// Errors
+//
+// The errors returned from this package can be inspected in several ways:
+//
+// The Code function from gocloud.dev/gcerrors will return an error code, also
+// defined in that package, when invoked on an error.
+//
+// The As function from the package golang.org/x/xerrors can retrieve the driver
```

Comment:
I would leave the `xerrors` thing off, as there's no guarantee that they will do the same thing.

Reply:
Done.

But this is just kicking the can down the road. When the new errors functionality lands in the standard library, we should conform to it. We can discuss another time.